lib/utils/validateOptions.mjs

```diff
@@ -36,7 +36,7 @@ export default function validateOptions(result, ruleName, ...optionDescriptions)
  * @param {(message: string) => void} complain
  */
 function validate({ possible, actual, optional }, ruleName, complain) {
-	if (actual === false && !ruleName.startsWith('report')) {
+	if (actual === false && !/^report[A-Z]/.test(ruleName)) {
```

Comment:
see https://github.com/stylelint/stylelint/pull/6250#discussion_r941493067

Reply:
[suggest] I'm not 100% sure, but can we explicitly specify `report*` rule names like this?

```js
const REPORT_RULE_NAMES = new Set(['reportDescriptionlessDisables', 'reportNeedlessDisables', 'reportInvalidScopeDisables']);
```

```suggestion
	if (actual === false && !REPORT_RULE_NAMES.has(ruleName)) {
```